File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/While/lista_fabio/AlgoritimosProgramacao/Algoritmos/Questões/List-01/q04.js
import { question } from "readline-sync";
function main(){
    //Input
    let dollar_real = parseFloat(question("Enter the dollar value converted in real:"))
    let dollar_value = parseInt(question("Enter the quantity of dollars: "))

    //Processing
    let total_value_dollar = dollar_real * dollar_value

    //Output
    console.log(`The total of the ${dollar_value} dollars converted to real is: ${total_value_dollar}$`)
}
main()

File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/Questões/List-01/q04.py
def main():
    #Input
    dollar_real = float(input("Enter the dollar value converted in real: "))
    dollar_value = int(input("Enter a dollar value: "))

    #Processing
    total_value_dollar = dollar_real * dollar_value

    #Output
    print(f"The total of the {dollar_value} dollars converted to real is equal to:  {total_value_dollar}$")

main()